### PR TITLE
solution

### PR DESCRIPTION
**Pull request by olegpaunov**

### CODE REVIEW

**Comment by olegpaunov** [comment]
nice task

File: src/index.html [hunk 1]
```diff
@@ -2,13 +2,68 @@
...
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo__image"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
```
Comment (Moroz-Dmytro) [inline src/index.html:29]:
```suggestion
            <a href="#apple" class="nav__link is-active">Apple</a>
```

Comment (Moroz-Dmytro) [reply src/index.html:29]: Fix all links

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,79 @@
...
+.nav__link {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+
+  height: 60px;
+
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
```
Comment (Moroz-Dmytro) [inline src/style.css:51]: Better will be to set this as default font styles for page, under html selector

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,79 @@
...
+  color: #000;
+
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active {
```
Comment (Moroz-Dmytro) [inline src/style.css:65]:
```suggestion
.is-active {
```

Comment (Moroz-Dmytro) [reply src/style.css:65]: image

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,79 @@
...
+  color: #00acdc;
```
Comment (Moroz-Dmytro) [inline src/style.css:66]: create variables for repeatable values

**Review by Moroz-Dmytro — CHANGES_REQUESTED** [review]
Nice progress 🔥 
But added some comments to improve your code
Also, try to fix this on smaller screens
image

**Pull request closed without merging**